caps/views.py

```diff
@@ -20,6 +20,12 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context['total_councils'] = Council.objects.all().count()
         context['percent_councils_with_plan'] = Council.percent_with_plan()
+        context['search_suggestions'] = [
+            '"electric taxis"',
+            '"green homes grant"',
+            'ground source heat pumps',
```

Comment:
I left the other two without double quotes because they seemed to bring up fairly sensible results without them. Also hints to people that they can use or not use double quotes, just like a Google search, but maybe that’s a bit subtle.